application/app/modules/BackendModule.java

```diff
@@ -3,28 +3,49 @@
 import com.google.inject.AbstractModule;
 import com.google.inject.Provides;
 import com.typesafe.config.Config;
-import event.EventRouter;
-import event.EventRouterImpl;
+import event.EventService;
+import event.EventServiceConfig;
+import event.EventServiceImpl;
+import event.router.EventRouterType;
 import mpd.MpdService;
 import mpd.MpdServiceImpl;
+import rfid.RfidTagUid;
 
-import javax.inject.Inject;
+import java.util.Map;
+import java.util.stream.Collectors;
 
 
 public class BackendModule extends AbstractModule {
-
     private final Config backendConfig;
 
-    @Inject
```

Comment:
Did this even work?

Reply:
That worked fine. Too many @Injects do not seem to be a problem...